fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -236,6 +236,7 @@ class SWIFT_CXX_IMMORTAL_SINGLETON_TYPE ServerKnobs : public KnobsImpl<ServerKno
 	// When the sampled read operations changes more than this threshold, the
 	// shard metrics will update immediately
 	int64_t SHARD_READ_OPS_CHANGE_THRESHOLD;
+	bool CHECK_WRITE_TRAFFIC_WHEN_EXIT_SPLIT; // experimental
```

Comment:
maybe rename something like `ENABLE_WRITE_BASED_SHARD_SPLIT`.

Reply:
Current shard split already support write traffic split. Not sure whether `ENABLE_WRITE_BASED_SHARD_SPLIT` can lead confusion.

Reply:
> maybe rename something like `ENABLE_WRITE_BASED_SHARD_SPLIT`.

Current shard split already support write traffic split. Not sure whether ENABLE_WRITE_BASED_SHARD_SPLIT can lead confusion.